docs/requirements.txt

```diff
@@ -1,3 +1,4 @@
 mkdocs>=1.6.0
 mdx_truly_sane_lists>=1.3
 pymdown-extensions>=10.8.1
```

Comment:
Captions were only introduced in v10.12 (https://github.com/facelessuser/pymdown-extensions/releases). You need to update the version here appropriately.